framework/tests/value/Basics.java

```diff
@@ -168,9 +168,9 @@ public void intCastTest(@IntVal({0, 1}) int input) {
         @IntVal({0, 1}) int c = (int) input;
         @IntVal({0, 1}) int ac = (@IntVal({0, 1}) int) input;
         @IntVal({0, 1, 2}) int sc = (@IntVal({0, 1, 2}) int) input;
-        // :: warning: (cast.unsafe)
+        // :: error: (cast.incomparable)
```

Comment:
Isn't this a cast from "0 or 1" to "definitely 1", so a downcast? Why is this incomparable now?